.eslintrc.json

```diff
@@ -4,7 +4,7 @@
     },
     "extends": "airbnb-base",
     "rules": {
-        "no-param-reassign": "warn",
+        "no-param-reassign": 0,
```

Comment:
would removing the entire rule works as well?

Reply:
Nope, removing the rule doesn't work as this rule is present in `airbnb-base` so causes errors, if we remove it.